src/__tests__/diode-test.js

```diff
@@ -1,6 +1,21 @@
 var Diode = require('../diode')
 
 describe('Diode', function() {
+  it ('is an event emitter', function() {
+    Diode.should.have.property('listen')
+  })
+
+  it ('is a decorator function', function() {
+    let emitter = Diode({ prop: 'yes' })
+
+    emitter.should.have.property('listen')
+    emitter.should.have.property('prop', 'yes')
+  })
+
+  it ('defaults to an empty object when decorating', function() {
+    let emitter = Diode()
+    emitter.should.have.property('listen')
```

Comment:
Really, I want to make sure it has the methods Diode has. I could do this better... Let's see